Mother2GbaTranslation.sln

```diff
@@ -0,0 +1,98 @@
+﻿
```

Comment:
It's unusual to associate source files with a solution. They should be associated with a project instead so that build settings, etc. can be included.

Reply:
Should I just remove the solution?

Reply:
Yeah I'd prefer that. You can leave the makefile though. If we build up the makefile then we can create a VS project to wrap it later.

Reply:
Done.